hcfb-be/set_config.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Are all of our developers on macOS/Linux?

This would be good to know for working on tooling moving forward :smile:

Reply:
I work mainly from my Windows desktop computer on this project unfortunately.

Reply:
@BenjaminWalkerBond No worries. Actually if it's Windows 10 you can run bash scripts anyway, right?

Otherwise we could always port to PowerShell.

Reply:
Added a batch script. I really hate Windows.